courses/level101/linux_basics/command_line_basics.md

```diff
@@ -394,7 +394,7 @@ grep <word_to_search> <file_name>
 In this example, we are trying to search for a string "1" in this file.
 The grep command outputs the lines where it found this string.
 
-![](images/linux/commands/image5.png)
+![](images/linux/commands/image35.png)
```

Comment:
This one still looks small, and is a different size from the image34. Can you make it consistent?

Reply:
I will try to make it consistent as well.